hack/hypernetes/README.md

```diff
@@ -0,0 +1,46 @@
+# Deployment scripts for hypernetes
+
+## System requirements
+
+The script is only tested on CentOS 7.2 with at least 8GB memory. It will install latest hypernetes with hyperd and OpenStack Mitaka.
```

Comment:
8GB is too much for most fresh new users. I think it also works in 4GB env. So how about 4GB at least and 8GB for recommend?